Dockerfile

```diff
@@ -1,3 +1,14 @@
+FROM golang:alpine
```

Comment:
Please name the build container using `AS builder`.

Reply:
Done